### DISCUSSION TITLE
تمپلیت های مختلف برای مرزبان

### DISCUSSION THREAD

**Discussion by ghost** [Gozargah/Marzban#545] [Show and tell]
لیستی از تمپلیت های شخصی سازی شده برای مرزبان

# لیست تمپلیت ها
- [تمپلیت برای clash و clash-meta](https://github.com/oXIIIo/marzban-template/tree/master/clash)
- [تمپلیت برای sing-box](https://github.com/oXIIIo/marzban-template/tree/master/singbox)
- [صفحه سابسکریپشن](https://github.com/oXIIIo/marzban-template/tree/master/subscription)
- [صفحه خانه](https://github.com/oXIIIo/marzban-template/tree/master/home)

# مراحل نصب
برای نصب هر تمپلیت به صفحه مربوط به آن مراجعه کنید

# نصب همه
برای نصب همه تمپلیت های موجو دستورات زیر را در ترمینال سرور خود اجرا کنید:
1. دانلود فایل های تمپلیت
```sh
sudo wget -N -P /var/lib/marzban/templates/clash/ https://raw.githubusercontent.com/oXIIIo/marzban-template/master/clash/default.yml
sudo wget -N -P /var/lib/marzban/templates/singbox/ https://raw.githubusercontent.com/oXIIIo/marzban-template/master/singbox/default.json
sudo wget -N -P /var/lib/marzban/templates/subscription/ https://raw.githubusercontent.com/oXIIIo/marzban-template/master/subscription/index.html
sudo wget -N -P /var/lib/marzban/templates/home/ https://raw.githubusercontent.com/oXIIIo/marzban-template/master/home/index.html
```
2. دستورات زیر رو تو ترمینال سرورتون بزنید:
```sh
echo 'CUSTOM_TEMPLATES_DIRECTORY="/var/lib/marzban/templates/"' | sudo tee -a /opt/marzban/.env
```
یا مقادیر زیر رو در فایل `.env` در پوشه `/opt/marzban` قرار بدین
```sh
CUSTOM_TEMPLATES_DIRECTORY="/var/lib/marzban/templates/"
```

3. ری استارت مرزبان
```sh
marzban restart
```

## بروزرسانی
برای بروزرسانی تمپلیت ها فقط کافیست مرحله 1 را تکرار کنید.

**Comment by abedin70** [comment]
من از هاست برای ساب استفاده میکنم.
همون کد پیشفرض مرزبان هست و اصلا تمپلت تغییر ندادم!
اگه اینو بزنم تو هاست هم اعمال میشه؟
یا باید کار خاصی انجام بدم؟
یا اصلا مشکلی پیش نمیاره!!؟

**Comment by ghost** [comment]
> اگه اینو بزنم تو هاست هم اعمال میشه؟ یا باید کار خاصی انجام بدم؟

از هاست استفاده نکردم اگر user-agent رو میفرسته و نتیجه رو به همون صورت نشون میده میتونی استفاده کنی.
> یا اصلا مشکلی پیش نمیاره!!؟

اگر user-agent نفرسته فایل کانفیگ مربوط به کلاینت نشون داده نمیشه ولی مشکلی خاصی ایجاد نمیشه فقط اون کلاینت ارور میده.
مثلا لینک سابسکریپشن روی کلش کار نمیکنه ولی روی v2rayng مشکلی نداره.

**Reply by abedin70** [reply]
جواب داد رو هاست 
ممنون

**Comment by rbsdotnet** [comment]
من از قالب صفحه home استفاده کردم 
تغییری ایجاد نشد
امکانش هست یه تصویر از قالب بزارید

**Reply by Eslender73** [reply]
پنلتو بدون داشبورد بز فقط اون‌تغییر میکنه

**Comment by Lucas02199** [comment]
سلام من کریت یوزر میکنم اون بالا مینویسه ساخته شده وای پابین یوزر ب من نشون نمیده چکار کنم

**Comment by Mjzaret** [comment]
من همه این تملیتها رو نصب کردم. حالا چطوری میتونم ازشون استفاده کنم؟
من بیشتر استفاده از Sing-Box برام مدنظر بود که میشد به صورت Json تنظیمات روتینگ رو به سمت کاربر فرستاد و سایتهای ایرانی مستقیم باز بشن
من از لینک سابسکریبشن استفاده میکنم.
وقتی ساب رو آپدیت میکنم هیچ تفاوتی نمکینه

**Reply by ghost** [reply]
سلام لینک سابسکریپشن رو تو کلاینت سینگ باکس بزنید میاره.

**Reply by Mjzaret** [reply]
تفاوتی با لینک ساب معمولی نداشت. همون اطلاعاتی رو برام آورد که با همین تنظیمات پیش فرض هست. به صورت json چیزی نداد که تنظیمات روتینگ داخلش باشه. میشه در این زمینه راهنمایی کنید.

**Reply by ghost** [reply]
اخر لینک سابسکریپشنتون /sing-box رو اضافه کنید برای مثال
https://example.com/sub/.../sing-box
فایل جیسون دانلود میشه اگر روتینگ ایران داخلش نباشه فایل تمپلیت رو به درستی نصب نکردین و یا مرزبان رو ری استارت نکردین.

**Comment by MehdiHakimi59** [comment]
سلام و خسته نباشید ... 
لینکها کار نمیکنن ، آیا آدرسها تغییر کردن ؟
لطفا بررسی فرمایید. باسپاس
